crates/code_generation/src/cyclers.rs

```diff
@@ -379,7 +379,27 @@ fn generate_consumer_identifiers(cyclers: &Cyclers) -> Vec<Ident> {
         .collect()
 }
 
-fn generate_start_method() -> TokenStream {
+fn generate_start_method(cycler_kind: CyclerKind) -> TokenStream {
+    let scheduler_tokens = match cycler_kind {
+        CyclerKind::Perception => quote! {
+            unsafe {
+                let priority = libc::sched_param {
+                    sched_priority: 10,
```

Comment:
This is 10 but the PR description mentions 20?

Reply:
hoppla, meant to write 10 in the description, that way it has lower priority compared to hula